theme/lumo/incubator-breadcrumb-styles.html

```diff
@@ -1,9 +1,22 @@
-<link rel="import" href="../../../vaadin-lumo-styles/color.html">
+<link rel="import" href="../../../vaadin-lumo-styles/typography.html">
 <link rel="import" href="../../../vaadin-lumo-styles/spacing.html">
+<link rel="import" href="../../../vaadin-lumo-styles/icons.html">
 
-<dom-module id="lumo-incubator-element" theme-for="incubator-breadcrumb">
+
+<dom-module id="lumo-incubator-breadcrumb" theme-for="incubator-breadcrumb">
   <template>
-    <style include="lumo-incubator-element-styles">
+    <style>
+      :host {
+        font-family: var(--lumo-font-family);
+      }
+
+      [part="separator"] {
+        margin: var(--lumo-space-wide-xs);
+      }
+
+      [part="separator"]::after {
+        content: '>';
```

Comment:
This should not be separated from the component I guess

Reply:
It's better that it come from theme. Something like a vaadin-icon for example. But for now I would settle for '>' only.

Reply:
@miladkdz, you should be able to see something when you import the component from the src folder.